python-package/lightgbm/callback.py

```diff
@@ -14,10 +14,13 @@ class EarlyStopException(Exception):
     ----------
     best_iteration : int
         The best iteration stopped.
+    best_score : float
```

Comment:
there might be multiple eval datasets and multiple eval metrics, you cannot use only one floating number for best_score